### DISCUSSION TITLE
displaying the current day on the screen

### DISCUSSION THREAD

**Discussion by belginunal** [jspsych/jsPsych#1410] [Q&A] [answered]
Hello, 

I am struggling to get the current day of the week and display it on the screen (along with a survey box). 

Specifically, what I need to do is to display the last 4 days on the screen (one at a time), and ask people to write a few things regarding what happened on that day. I want to display the day of the week so that participants will know which day they should report about.
I am getting very confused and wondered if someone has a ready fix to this sort of thing.

I would really appreciate any advice. Thanks for reading this!

**Comment by becky-gilbert** [comment, marked as answer]
Hi @belginunal, have you tried using the JavaScript [`Date.getDay` function](https://www.w3schools.com/jsref/jsref_getday.asp)? This function returns a number, where 0 = Sunday, 1 = Monday, etc. You could use that along with an array of the day names to get the current day name, and then subtract from the current day index to get the indices/names of the last 4 days. Feel free to post your code if you need further guidance.

**Reply by belginunal** [reply]
Thank you so much!
